web/client/src/util/iterate.js

```diff
@@ -0,0 +1,33 @@
+export function* iterate(emscriptenList) {
```

Comment:
This file (and every other current JS file) is not really testable and thus not really convertable (as of right now) to TS, as the functionality using this does not work (as discussed, this will leave the Webclient in a half-converted state, but that is okay [discussed with AG]).